frontend/src/components/ShootDetails/ShootJournalsCard.vue

```diff
@@ -86,7 +89,9 @@ export default {
 
       const createdAt = `**Created At:** ${this.shootCreatedAt}`
       const lastOperation = `**Last Op:** ${get(this.shootLastOperation, 'description', '')}`
-      const lastError = `**Last Error:** ${get(this.shootLastError, 'description', '-')}`
+      let shootLastErrorDescriptions = compact(map(this.shootLastErrors, 'description'))
+      shootLastErrorDescriptions = join(shootLastErrorDescriptions, '\n')
+      const lastError = `**Last Errors:** ${shootLastErrorDescriptions || '-'}`
```

Comment:
have you checked how the endresult looks? Can each error easily be recognized? Or should we use bulleted list / add seperator lines or similar

Reply:
I don't know actually. I did not find any shoot with more than one error. I talked to the Gardener Team and they told me that usually there are multiple errors only if simultaneous operations fail, however I could not find such a situation. In general - yes, it would probably make sense to enhance our view here. The general look of this dialog is not very good as we show lengthy error messages. Adding separators could on the one hand improve readability. On the other hand it would also increase required space and could eventually make the dialog too big for the screen (which can happen anyway, depending on the amount and length of the messages). As I did not observe this happening too often, I decided to not invest here atm.